docs/01. set up database

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE chitter_test;
+CREATE TABLE peeps (id SERIAL PRIMARY KEY, message VARCHAR(255), time TIMESTAMP DEFAULT CURRENT_TIMESTAMP);
```

Comment:
is your database timestamping itself?
:) very clever way to achieve this. I will need to read up on how to do this.